api/src/main/java/io/opentelemetry/trace/Sampler.java

```diff
@@ -58,4 +59,29 @@ boolean shouldSample(
    * @since 0.1.0
    */
   String getDescription();
+
+  /**
+   * Sampling decision returned by {@link Sampler#shouldSample(SpanContext, Boolean, TraceId,
+   * SpanId, String, List)}.
+   *
+   * @since 0.1.0
+   */
+  interface SamplingDecision {
```

Comment:
Since this is already in the context of Sampler, I'd suggest removing the `Sampling` part of the name, resulting in just `Decision`.

Reply:
+1